pkg/controller/loadtest.go

```diff
@@ -306,9 +310,9 @@ func (c *Controller) syncHandler(key string) error {
 		reportURL = fmt.Sprintf("%s/load-test/%s/report", c.cfg.KangalProxyURL, loadTest.GetName())
 	}
 
-	backend, err := backends.NewLoadTest(loadTest, c.kubeClientSet, c.kangalClientSet, c.logger, c.namespacesLister, reportURL, c.cfg.PodAnnotations, c.cfg.NamespaceAnnotations, c.cfg.Backends)
+	resolved, err := c.registry.Resolve(loadTest.Spec.Type)
```

Comment:
For some reason the `registry` or `Resolve` name here seems off to me.
Based on [Resolve](https://github.com/hellofresh/kangal/blob/ddb64691658d562862ebb6a7e29d2defd5801718/pkg/backends/internal/registry.go#L61) I think `Get` or `GetBackend` maybe a better name here.